### PR TITLE
Dev containers blog

### PR DESCRIPTION
**Pull request by bamurtaugh**
Here's the blog about the [dev container CLI](https://github.com/devcontainers/spec/issues/9) so that we can get some initial user feedback. 

A couple outstanding questions:
- I was thinking a publish tomorrow or Thursday (5/18 or 5/19) would be great - @gregvanl please let us know if this is feasible or you have a preference
- Should we consider removing the steps to build from sources, and just mention the `npm` install to keep the post more concise?

cc @Chuxel @jkeech @joshspicer @edgonmsft

### CODE REVIEW

File: blogs/2022/05/18/dev-container-cli.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+---
+
+# The dev container CLI
+**A consistent environment anywhere with the dev container CLI**
+
+May 18, 2022 by Brigit Murtaugh, [@BrigitMurtaugh](https://twitter.com/BrigitMurtaugh)
+
+## TLDR;
+
+We’ve released a dev container CLI as part of the new Development Containers Specification. We’d love your feedback in [this issue](https://github.com/devcontainers/cli/issues/7), or new issues and PRs in the repo.
```
Comment (gregvanl) [inline blogs/2022/05/18/dev-container-cli.md:17]: Should we be clearer who "we" is? Is it the VS Code team, Microsoft, or a dev container consortium or community?

File: blogs/2022/05/18/dev-container-cli.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## Context
```
Comment (gregvanl) [inline blogs/2022/05/18/dev-container-cli.md:19]: Maybe a less generic header? How about "Consistency = joy"?
Or if that is too poppy, "A consistent, predictable environment"

File: blogs/2022/05/18/dev-container-cli.md (hunk 1 above)
Comment (bamurtaugh) [reply blogs/2022/05/18/dev-container-cli.md:17]: Hmm that's a good point. We want to emphasize dev containers are an open spec for any user or tool to adopt, similar to LSP. In the [LSP announcement blog](https://code.visualstudio.com/blogs/2016/06/27/common-language-protocol), it starts with mentioning VS Code and then moves into using "we." 

Maybe we say "Our dev containers team at Microsoft and GitHub is excited to release a dev container CLI..."? How does that sound?

File: blogs/2022/05/18/dev-container-cli.md (hunk 2 above)
Comment (bamurtaugh) [reply blogs/2022/05/18/dev-container-cli.md:19]: I like "Consistency = joy," it made me smile when I read that! I'll update.

File: blogs/2022/05/18/dev-container-cli.md (hunk 1 above)
Comment (gregvanl) [reply blogs/2022/05/18/dev-container-cli.md:17]: Are there more than one team across the two companies?
"The dev container teams at Microsoft and GitHub are excited to release a dev container CLI..."

Comment (bamurtaugh) [reply blogs/2022/05/18/dev-container-cli.md:17]: Yeah I suppose we could consider them multiple teams. Updated to this wording.

Comment (bamurtaugh) [reply blogs/2022/05/18/dev-container-cli.md]: Hmm looking at it now, we say "dev containers" 3 times in this sentence 😄 Do you think "The dev container teams at Microsoft and GitHub are excited to release a **new** CLI as part of the new Development Containers Specification" makes sense?

[the remark this answers is not in the document]
Comment (bamurtaugh) [reply blogs/2022/05/18/dev-container-cli.md]: We have this repo: https://github.com/devcontainers/spec. I'll update the link to that.

File: blogs/2022/05/18/dev-container-cli.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+---
+
+# The dev container CLI
+**A consistent environment anywhere with the dev container CLI**
+
+May 18, 2022 by Brigit Murtaugh, [@BrigitMurtaugh](https://twitter.com/BrigitMurtaugh)
+
+## TLDR;
+
+The dev container teams at Microsoft and GitHub are excited to release a CLI as part of the new [Development Containers Specification](https://github.com/devcontainers/spec). We’d love your feedback in [this issue](https://github.com/devcontainers/cli/issues/7), or new issues and PRs in the repo.
```
Comment (Chuxel) [inline blogs/2022/05/18/dev-container-cli.md:17]: Maybe mention it's an open source CLI?

File: blogs/2022/05/18/dev-container-cli.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+{"outcome":"success"}
+```
+
+Congrats, you’ve just run the dev container CLI and seen it in action!
+
+These steps are also provided in the CLI repo’s [readme](https://github.com/devcontainers/cli/blob/main/README.md).
+
+## How can I get involved?
+
+Your feedback is incredibly helpful in shaping and expanding the spec, and we’d love your feedback on the CLI so far. Please check out the CLI and let us know what you think. We’ve [opened an issue](https://github.com/devcontainers/cli/issues/7) specifically for folks to leave comments and questions, and you can also certainly also open new issues or PRs in the repo, just like other open-source projects.
```
Comment (Chuxel) [inline blogs/2022/05/18/dev-container-cli.md:123]: Would we also want to mention that people can provide feedback on the spec with a link to that repo's issues?

**Review by Chuxel — APPROVED** [review]
Made a couple small suggestions, but otherwise looks great!

File: blogs/2022/05/18/dev-container-cli.md (hunk 3 above)
Comment (bamurtaugh) [reply blogs/2022/05/18/dev-container-cli.md:17]: Great point, added "release an **open source** CLI..."

File: blogs/2022/05/18/dev-container-cli.md (hunk 4 above)
Comment (bamurtaugh) [reply blogs/2022/05/18/dev-container-cli.md:123]: That's a good idea. Added:

> We welcome your [feedback on the spec](https://github.com/devcontainers/spec/issues) anytime, and we’ll continue providing updates and info about what you can try out and when.

**Review by chrmarti — APPROVED** [review]
Reads great!

**Pull request merged**